src/Condition.cpp

```diff
@@ -143,14 +145,21 @@ void ePhotosynthesis::readFile(const std::string &filename, std::map<std::string
         if (input.empty())
             continue;
         boost::algorithm::split_regex(tempVec, input, token);
+        existing = mapper.find(tempVec[0]);
+        if (existing != mapper.end()) {
+          std::cerr << "readFile[" << filename << "]: " << tempVec[0] <<
+            " already set. Using previous value." << std::endl;
+          continue;
+        }
         mapper.insert(std::pair<std::string, std::string>(tempVec[0], tempVec[1]));
     }
 }
 
-void ePhotosynthesis::readFile(const std::string &filename, std::map<std::string, double> &mapper) {
+void ePhotosynthesis::readFile(const std::string &filename, std::map<std::string, double> &mapper, bool enzymes) {
```

Comment:
When using the readFile function, including the argument `enzymes` and making it `true` is required to change Vmax's units.